lib/reek/smells/unused_private_method.rb

```diff
@@ -0,0 +1,82 @@
+require_relative 'smell_detector'
+require_relative 'smell_warning'
+require_relative '../context/method_context'
+
+module Reek
+  module Smells
+    # Class for storing `hits` which are unused private methods
+    # we found in the given context. `name` and `line` are then used to
+    # construct SmellWarnings.
+    class Hit
```

Comment:
Maybe this class should be moved into the smell detector class, since it is only used there?